ldap/servers/plugins/collation/orfilter.c

```diff
@@ -901,7 +901,7 @@ or_filter_index(Slapi_PBlock *pb)
 {
     auto or_filter_t * or = or_filter_get(pb);
     auto int rc = LDAP_UNAVAILABLE_CRITICAL_EXTENSION;
-    auto IFP mrINDEX_FN = NULL;
+    int32_t (*mrINDEX_FN)(Slapi_PBlock *) = NULL;
     auto struct berval **mrVALUES = NULL;
     auto char *mrOID = NULL;
     auto int mrQUERY_OPERATOR;
```

Comment:
While we are here, should we remove `auto`? It's useless, as far as I know.

Reply:
> While we are here, should we remove `auto` in other places too? It's useless in modern C, as far as I know.

Fixed